app/assets/stylesheets/taxs.css

```diff
@@ -276,8 +276,6 @@ main, section, article, aside {
 #wrapper #content .page-header h1 {
   clear: right; }
 
-.selectYear#wrapper #content .page-header #user-info {
-  float: none;}
```

Comment:
Is this meant to be deleted?

Reply:
Yes, it was causing formatting issues with the link appearing below user name, utr rather than along side it.